### PR TITLE
Fix check gossiper generation

### PR DESCRIPTION
**Pull request by vladzcloudius**
* Remove the dependency on the scylla.yaml contents.
* Check all cluster's nodes at once.
* Add the framework to provide parameters.

### CODE REVIEW

**Comment by vladzcloudius** [comment]
@tarzanek Could you, please, review?

File: scylla-check-gossiper-generation/scylla-check-gossiper-generation.sh [hunk 1]
```diff
@@ -2,31 +2,62 @@
 #
 # This script checks the node's gossiper generation and produces an alert if it is older 
 # than 1 year.
+# !
+# !scylla-check-gossiper-generation.sh [-h] [-n <nodetool command, by default 'nodetool'>]
+# !
 
-set -e
+usage() {
+    cat $0 | grep "^# !" | cut -d"!" -f2-
```
Comment (tarzanek) [inline scylla-check-gossiper-generation/scylla-check-gossiper-generation.sh:10]: wouldn't it be easier just to echo the single line? ;-)
but this works too, so just nitpicking

**Review by tarzanek — APPROVED** [review]
lgtm, if this functionally works/was tested
then I don't see a problem
logically it makes sense and properly checks the whole ring instead of single IP

**Comment by tarzanek** [comment]
merging, thnx Vlad

**Pull request merged**